testsuite/integration/basic/src/test/java/org/wildfly/test/integration/observability/opentelemetry/BasicOpenTelemetryTestCase.java

```diff
@@ -19,87 +19,51 @@
-public class BasicOpenTelemetryTestCase {
+public class BasicOpenTelemetryTestCase extends BaseOpenTelemetryTest {
     @Inject
     private Tracer tracer;
 
     @Inject
     private OpenTelemetry openTelemetry;
 
-    private static final String WEB_XML
-            = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
-            + "<web-app xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns=\"http://java.sun.com/xml/ns/javaee\"\n"
-            + "         xsi:schemaLocation=\"http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd\"\n"
-            + "         metadata-complete=\"false\" version=\"3.0\">\n"
-            + "    <servlet-mapping>\n"
-            + "        <servlet-name>jakarta.ws.rs.core.Application</servlet-name>\n"
-            + "        <url-pattern>/*</url-pattern>\n"
-            + "    </servlet-mapping>"
-            + "</web-app>";
-
-    @Deployment
-    public static Archive<?> deploy() {
-        return ShrinkWrap.create(WebArchive.class, BasicOpenTelemetryTestCase.class.getSimpleName() + ".war")
-                .addClasses(OpenTelemetrySetupTask.class, ServerSetupTask.class)
-                .addAsWebInfResource(new StringAsset(WEB_XML), "web.xml")
-                .addAsWebInfResource(EmptyAsset.INSTANCE, "beans.xml")
-                .addAsManifestResource(createPermissionsXmlAsset(
-                        // Required for the client to connect
-                        new SocketPermission(TestSuiteEnvironment.getHttpAddress() + ":" +
-                                TestSuiteEnvironment.getHttpPort(), "connect,resolve"),
-                        new SecurityPermission("insertProvider"),
-                        new MBeanServerPermission("createMBeanServer"),
-                        new MBeanPermission("*", "registerMBean, unregisterMBean, invoke"),
-                        new RuntimePermission("getClassLoader"),
-                        new RuntimePermission("modifyThread"),
-                        new RuntimePermission("setContextClassLoader")
-                ), "permissions.xml");
-    }
+    @Inject
+    private Baggage baggage;
 
     @Test
-    public void hasDefaultInjectedOpenTelemetry() {
+    public void openTelemetryInjection() {
         Assert.assertNotNull(openTelemetry);
     }
 
     @Test
-    public void hasDefaultInjectedTracer() {
+    public void traceInjection() {
         Assert.assertNotNull(tracer);
     }
 
+    @Test
+    public void baggageInjection() {
+        Assert.assertNotNull(baggage);
+    }
     @Test
     public void restClientHasFilterAdded() {
-        Client client = ClientBuilder.newClient();
-        org.wildfly.common.Assert.assertTrue(client.getConfiguration().getClasses().stream()
-                .map(c -> c.getCanonicalName())
-                .anyMatch(n -> "org.wildfly.extension.opentelemetry.api.OpenTelemetryClientRequestFilter".equals(n)));
+        org.wildfly.common.Assert.assertTrue(
```

Comment:
Is ```org.wildfly.common.Assert``` intended here? (Test failure vs. error?)

Reply:
It was not. Fixed.